src/app/(bundle)/bundle/onboarding/BundleOnboardingView.module.scss

```diff
@@ -105,24 +112,27 @@
       }
 
       .logoWrapper {
-        width: 50px;
-        height: 50px;
+        height: 64px;
         display: flex;
         justify-content: center;
         align-items: center;
 
         img {
           width: 100%;
+          height: 100%;
+          object-fit: cover;
         }
       }
 
       h2 {
+        margin-top: 12px; // hardcoding this because we currently don't have a 12px token. TODO: MNTOR-4552
```

Comment:
TBH if the set of tokens isn't used by our designers, then I'm not really sure what the point of having them is anyway :melting_face: Which is to say: ah well, I guess hardcoding is fine.

However, I do think we should (and have mostly done so far) avoid having element styles affect elements outside of their own box, i.e. avoid margins. This makes layout more predictable and helps avoid future elements accidentally affecting the layout of these. Instead, ideally we let element's layout and positioning be handled by their parents, i.e. wrap it in a column flexbox, and set a `gap` and `padding` on that.

Reply:
Yeah for the tokens that sounds fine, but we can still wrap this in a `flex` element and use `gap` instead of `margin-top`, right?